packages/terra-clinical-application/CHANGELOG.md

```diff
@@ -0,0 +1,38 @@
+ChangeLog
+=========
+
+Unreleased
+----------
+
```

Comment:
We should add react 16 to the change log

Reply:
good call